src/GovUk.Education.ExploreEducationStatistics.Content.Search.FunctionApp.Tests/Functions/OnReleaseVersionPublished/OnReleaseVersionPublishedFunctionTests.cs

```diff
@@ -34,21 +31,78 @@ public async Task GivenEvent_WhenPayloadContainsPublicationSlug_ThenRefreshSearc
         var response = await sut.OnReleaseVersionPublished(eventGridEvent, new FunctionContextMockBuilder().Build());
         
         // ASSERT
-        var actual = Assert.Single(response);
+        Assert.NotNull(response);
+        var actual = Assert.Single(response.RefreshSearchableDocumentMessages);
```

Comment:
Might be clearer if the method name changed to something other than `_ThenRefreshSearchableDocumentMessageDtoReturned`, because the return type is now `OnReleaseVersionPublishedOutput`?

Something like `_ThenSearchableDocumentIsRefreshed`?